packages/jellyfish-address/tsconfig.json

```diff
@@ -1,10 +1,3 @@
 {
-  "extends": "../../tsconfig.json",
-  "include": [
-    "./src/**/*"
-  ],
-  "compilerOptions": {
-    "outDir": "./dist",
-    "rootDir": "./src",
-  }
+  "extends": "../../tsconfig.json"
```

Comment:
do we still need this file, since we are just extending the root config without overriding anything?

Reply:
ah yea.. we don't need that